python/lsst/cp/pipe/defects.py

```diff
@@ -238,10 +253,41 @@ def findHotAndColdPixels(self, exp, nSigma):
                 continue
 
             # Remove a background estimate
-            ampImg -= afwMath.makeStatistics(ampImg, afwMath.MEANCLIP, ).getValue()
+            meanClip = afwMath.makeStatistics(ampImg, afwMath.MEANCLIP, ).getValue()
+            ampImg -= meanClip
 
+            # Determine thresholds
+            stDev = afwMath.makeStatistics(ampImg, afwMath.STDEVCLIP, ).getValue()
+            expTime = exp.getInfo().getVisitInfo().getExposureTime()
+            thresholdType = self.config.thresholdType
+            datasetType = exp.getMetadata().get('IMGTYPE', 'UNKNOWN')
+            if thresholdType == 'VALUE':
+                hotPixelThreshold = self.config.darkCurrentThreshold*expTime/amp.getGain()
```

Comment:
I would think that in the `value` land, the hot pixel threshold is for darks only, and the cold pixel threshold is for flats only.  (This is certainly what was done by EOtest in the past, and what has been vetted so far.)

Reply:
Correct. But in standard deviation land here in DM and as we have been doing so far, we do have cold and dark pixels from flats. We were discussing this with @czwa, and decided to keep value and stdev lands doing analogous things here in DM. Of course, then this would become a difference in definition with `eoTest` and I have pointed this out to Jim.   
I think we can make a decision here at this point. If we decide to be completely analogous to `eoTest` then we would need to do hot pixels for darks and only cold for flats, both in VALUE and STDEV modes.

Reply:
I can make a config option for this .

Reply:
I think a config option so we can run in both modes (cold pixels only from flats, or cold and hot both from flats) is the least confusing options.

Reply:
I added the config option.